### ISSUE TITLE
Questions for homework 8: capstone

### ISSUE THREAD

**Issue by k8hertweck** [fredhutchio/tfcb_2019#42]
Let me know what questions you have about the capstone project here!

**Comment by stephen-rettie** [comment]
For the report is it okay to have a markdown report that summarizes the findings and points to jupyter notebooks (commented) that generated figures or data? 

Or do I need to have a single jupyter notebook with all of the code and markdown?

**Comment by LABrumage** [comment]
I know based on lecture 03 that it is considered best practice to have a README within the data subdirectory as well, but for the purposes of this project is it acceptable to just have one README.md in the main directory that includes data documentation (among the other topics required according to the homework instructions)?

**Comment by k8hertweck** [comment]
@stephen-rettie The very end of the project description addresses this: "In addition to your report, you may (but are not required to) include additional files used for analysis (either as intermediate results, or scripts)." The report has required components, and is basically the main document for grading, but you can certainly have other files used in creating that final report.

**Comment by k8hertweck** [comment]
@LABrumage Yes, this is acceptable, especially since you are commenting on the data in your report as well.
